### DISCUSSION TITLE
New instrument pictures

### DISCUSSION THREAD

**Discussion by reinhardwh** [jamulussoftware/jamulus#1206] [UI Related]
Hi,
please add female/non-gender/.. profile pics
Thanks Reinhard

**Comment by ann0see** [comment]
I think there's already a ticket on that. I'd also support a general redesign of all the pictures. @geheimerEichkater and @mulyaj I think it would be great if we started a graphical redesign

**Comment by ann0see** [comment]
BTW, can I change the title to "new instrument pictures" (we do already have male & female pictures)?

**Reply by reinhardwh** [reply]
Ok 4 me :)

**Reply by ann0see** [reply]
Thanks!

**Comment by ann0see** [comment]
It would be great if we had this in the next release. Meanwhile I'd suggest you subscribe to https://github.com/jamulussoftware/jamulus/issues/899 and comment there.
